### PR TITLE
Fixed spectrum hover in skyportal/plot.py

### PR DESCRIPTION
**Pull request by scizen9**
Changed line plot style for spectra from step to line to enable hover tool.  Modified tooltips struct to access wavelength and flux and formatted same.

### CODE REVIEW

File: skyportal/plot.py [hunk 1]
```diff
@@ -783,12 +783,8 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, height=300):
...
     model_dict = {}
     for i, (key, df) in enumerate(split):
-        model_dict['s' + str(i)] = plot.step(
-            x='wavelength',
-            y='flux',
-            color=color_map[key],
-            source=ColumnDataSource(df),
-            mode="center",
+        model_dict['s' + str(i)] = plot.line(
+            x='wavelength', y='flux', color=color_map[key], source=ColumnDataSource(df),
```
Comment (dannygoldstein) [inline skyportal/plot.py:787]: we need `step` to properly show the spectra, otherwise they end up looking weird.

Comment (dannygoldstein) [reply skyportal/plot.py:787]: this was requested in https://github.com/fritz-marshal/fritz/issues/111

**Comment by scizen9** [comment]
Found a way to preserve step plot and enable hover tool.  I simply overplot with line, but with line_alpha=0.  Now all you see is the step plot, but the hover tool actually works.

File: skyportal/plot.py (hunk 1 above)
Comment (scizen9) [reply skyportal/plot.py]: See latest commits that keep step plots.

**Pull request merged**